.github/workflows/build_release.yml

```diff
@@ -0,0 +1,87 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Publish canary image to GitHub Container Registry
+
+on:
+  push:
+    tags: [ "v[0-9].[0-9].[0-9]" ]
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build_operator:
+
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Build and push operator image
+        uses: docker/build-push-action@v1.1.1
+        with:
+          # Username used to log in to a Docker registry. If not set then no login will occur
+          username: ${{ secrets.PACKAGES_USERNAME }}
+          # Password or personal access token used to log in to a Docker registry. If not set then no login will occur
+          password: ${{ secrets.PACKAGES_PASSWORD }}
+          # Server address of Docker registry. If not set then will default to Docker Hub
+          registry: docker.pkg.github.com
+          # Docker repository to tag the image with
+          repository: ${{ secrets.PACKAGES_USERNAME }}/http-add-on/operator
+          # Automatically tags the built image with the git SHA
+          tag_with_sha: true
```

Comment:
This is actually optional, I also tagged with the sha hash because we would have both the sha-hashed version and the release tag, but I'll change it in no time